Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1

```diff
@@ -213,6 +213,125 @@ Describe "BqJob-Copy" {
     }
 }
 
+Describe "BqJob-EL" {
+
+    BeforeAll {
+        $r = Get-Random
+        $datasetName = "pshell_testing_$r"
+        $test_set = New-BqDataset $datasetName
+        $folder = Get-Location
+        $folder = $folder.ToString()
+        $filename = "$folder\classics.csv"
+        $table = New-BqTable -Dataset $test_Set "table_$r"
+        New-BqSchema -Name "Title" -Type "STRING" | New-BqSchema -Name "Author" -Type "STRING" |
+            New-BqSchema -Name "Year" -Type "INTEGER" | Set-BqSchema $table | 
+            Add-BqTabledata $filename CSV -SkipLeadingRows 1
+        $bucket = New-GcsBucket "ps_test_$r"
+        $gcspath = "gs://ps_test_$r"
+    }
+
+    It "should set Load parameters correctly" {
```

Comment:
I think it would be nice to have a test that also checks defaults are properly set too. (and below)